konrad/convection.py

```diff
@@ -41,9 +45,26 @@ def energy_difference(T_2, T_1, sst_2, sst_1, phlev, eff_Cp_s):
     dT = T_2 - T_1  # convective temperature change of atmosphere
     dT_s = sst_2 - sst_1  # of surface
 
-    termdiff = - np.sum(Cp/g * dT * np.diff(phlev)) + eff_Cp_s * dT_s
+    term_diff = - np.sum(Cp/g * dT * np.diff(phlev)) + eff_Cp_s * dT_s
 
-    return termdiff
+    return term_diff
+
+
+def latent_heat_difference(h2o_2, h2o_1):
+    """Calculate the difference in energy from latent heating between two
```

Comment:
I am not sure how this will look like in the docs as it may simply use the first line only and cut off the remainder.